### PR TITLE
Fonts + Social

### PR DESCRIPTION
**Pull request by rosano**
Using some variation in the font weights/sizes so that it's not all at the same level + adding social images for sharing.

### CODE REVIEW

**Comment by rosano** [comment]
@hpk42 maybe our cosmos can be shared on social channels to let people know it exists and also encourage them to use for their own projects. Could be something like

> The Delta Chat cosmos is expanding. If your project is too, maybe you'd like to fork this and use it? https://cosmos.delta.chat

File: index.html [hunk 1]
```diff
@@ -13,14 +13,22 @@
...
+
 <style>
 body {
   margin: 0px;
   padding: 0px;
   font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
   background-color: #000;
   color: #bbb;
-  font-size: 16px;
+  font-size: 15px;
```
Comment (r10s) [inline index.html:31]: i would stay with the larger option of 16px here, this is at least better for my older eyes :)

but no blocker, also 15px is okayish :)

**Review by r10s — APPROVED** [review]
nice, much better!

having the links smaller is definitely an improvement and makes action wrt the "git"-link discussion as of https://github.com/deltachat/cosmos/pull/7#issuecomment-1357439327 maybe superfluous, at least it mitigates the issue.

File: index.html (hunk 1 above)
Comment (rosano) [reply index.html:31]: For me it goes a little too wide (sometimes it's a quirk of certain pixel values, like if you observe between 10px to 12px how it changes). If it's not a blocker, I'll keep it.

**Pull request merged**